book/index.adoc

```diff
@@ -18,6 +18,9 @@
 :description: HTML meta tag content for description
 :keywords: HTML meta tag content for keywords
 :icons: font
+:pdf-themesdir: {docdir}/themes
+:pdf-fontsdir: {docdir}/fonts
+:pdf-theme: basic
```

Comment:
let's keep this as an argument so that we can control that from the scripts. otherwise, if there are other themed versions then this needs to be edited.

Reply:
Imagine you actually have two themes: "basic" and "advanced". If I wanted to switch through them to create the PDF I'd have to manually update the config file. While in the other way I could just update my command (which is more accessible when running it via a CI, for example).

I didn't realize that command line arguments will override the asciidoc settings and if that's the case then we can leave the PR changes mostly as is (I'll revert the shell scripts changes though to retain that information as part of code comments in it).

Reply:
Makes sense.